README.md

```diff
@@ -1 +1,134 @@
+                        /etc/opt/senzing/G2Module.ini
+  -o OUTPUT_FILE_ROOT, --output_file_root OUTPUT_FILE_ROOT
+                        root name for files created such as
+                        "/project/snapshots/snapshot1"
+  -s SAMPLE_SIZE, --sample_size SAMPLE_SIZE
+                        defaults to 1000
+  -f RELATIONSHIP_FILTER, --relationship_filter RELATIONSHIP_FILTER
+                        filter options 1=No Relationships, 2=Include possible
+                        matches, 3=Include possibly related and disclosed.
+                        Defaults to 3
+  -a, --for_audit       export csv file for audit
+  -D, --debug           print debug statements
+```
+*the following are not in G2Snapshot-api-only*
+```console
+  -d DATASOURCE_FILTER, --datasource_filter DATASOURCE_FILTER
+                        data source code to analayze, defaults to all
+  -k CHUNK_SIZE, --chunk_size CHUNK_SIZE
+                        defaults to 1000000
+  -t THREAD_COUNT, --thread_count THREAD_COUNT
+                        defaults to 0
+  -u, --use_api         use api instead of sql to get resume
+```
+
+## Contents
+
+1. [Prerequisites](#Prerequisites)
+2. [Installation](#Installation)
+3. [Typical use](#Typical-use)
+
+### Prerequisites
+- Python 3.6 or higher
+- Senzing API version 1.10 or higher
+- database support for your Senzing database
+    - psycopg2 (for postgres)
+    - pyodbc (and supporting drivers) for all other databases 
+
+*If using an SSHD container, you should first max it out with at least 4 processors and 30g of ram as the more threads you give it, the faster it will run.  Also, if the database container is not set to 
+auto-scale, you should give it a additional resources as well.*
+
+### Installation
+
+1. Place the the following files in a directory of your choice:
+    - [G2Snapshot.py](G2Snapshot.py) 
+    - [G2Snapshot-api-only.py](G2Snapshot-api-only.py) 
+
+2. Set PYTHONPATH environment variable to python directory where you installed Senzing.
+    - Example: export PYTHONPATH=/opt/senzing/g2/python
+
+3. The senzing environment must be set to your project by sourceing the setupEnv script created for it.
+
+Its a good idea to place these settings in your .bashrc file to make sure the enviroment is always setup and ready to go.
+*These will already be set if you are using a Senzing docker image such as the sshd or console.*
+
+### Typical use
+
+To run for all data sources:
+```console
+python3 G2Snapshot.py -o /myproject/snapshots/snapshot-mm-dd-yyyy 
+```
+This will result in the following files being generated ...
+- /myproject/snapshots/snapshot-mm-dd-yyyy.json
+- /myproject/snapshots/snapshot-mm-dd-yyyy.csv (if you use -a for_audit option)
+
+*Of course you will want to change the -o output_path to a file and directory of your choice.  It is a good idea though to create a snapshots directory and include the date you took the snapshot in your file name 
```

Comment:
Modify the path for the -o (output_path) to your own choosing.

Reply:
not done, would have to do everywhere.